src/docfx/lib/process/ProcessUtility.cs

```diff
@@ -40,12 +40,12 @@ public static string Execute(string fileName, string commandLineArgs, string? cw
                 Task.Run(() => process.StandardError.BaseStream.CopyTo(Console.OpenStandardOutput()));
 
                 var result = stdout ? process.StandardOutput.ReadToEnd() : "";
-
+                var error = process.StandardError.ReadToEnd();
```

Comment:
The `stderr` is already piped and processed in line 40, if you need to access `stderr` content, do it inside the callback of line 40.